test/integration/get-path-and-run-tests.ts

```diff
@@ -25,9 +24,9 @@ async function getBuiltStudioPath() {
     .slice(0, -1);
 }
 
-async function execCommand(command) {
-  return new Promise((resolve, reject) => {
-    exec(command, (err, stdin, stderr) => {
+async function execCommand(command): Promise<ExecException & string> {
```

Comment:
Ist das `&` wirklich richtig hier? 

Hätte das hier erwartet:
```suggestion
async function execCommand(command): Promise<ExecException | string> {
```

Siehe hier:
https://www.tutorialspoint.com/typescript/typescript_bitwise_operators_examples.htm

Reply:
Sollte hier nicht Promise<string> der richtige Typ sein?
Die `ExecException` wird ja nicht resolved, sonden nur rejected.